packages/test/test-service-load/testConfig.json

```diff
@@ -115,6 +115,17 @@
                     }
                 }
             }
+        },
+        "scale_with_signals":  {
+            "opRatePerMin":  7,
+            "signalsPerMin":  46,
```

Comment:
can you add this to the ci profile above as well, so that we run it too. i think just matching the ops rate should be fine

Reply:
Done!